vllm/utils/gc_utils.py

```diff
@@ -89,6 +89,25 @@ def handle(self, phase: str, info: dict[str, int]) -> None:
             )
 
 
+def gc_freeze() -> None:
```

Comment:
Suggested rename for clarity
```suggestion
def freeze_gc_heap() -> None:
```

Reply:
Updated.